contributing/code_of_conduct/enforcement_team.rst

```diff
@@ -0,0 +1,58 @@
+Enforcement team
+================
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, the enforcement team
+pledge to ensure that the spirit of the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
+is respected. Our main priority is to ensure the safety of our community members.
+The second goal is to help educate the community as a whole to be aware of the CoC
+and how to help implement its spirit throughout the community. In case these goals
+conflict, we will prioritize safety of community members over all other goals.
+
+If you think there is or has been a violation to the code of conduct please contact
+enforcement team or if you prefer contact only individual members of the enforcement team.
+
+Members
+-------
+
+Here are all the members of the Code of Conduct enforcement team. You can contact
+any of them by sending an email or tweet. You could also contact all of them at once
+by emailing **coc@symfony.com**.
+
+Lukas Kahwe Smith
+`````````````````
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, susciit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+Lukas Kahwe Smith
+`````````````````
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, suscipit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+Lukas Kahwe Smith
+`````````````````
+
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, suscipit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+About the Enforcement Team
```

Comment:
A final question: the Symfony leader appoints Enforcement Team members ... but for ho long? Do these appointments expire? Do they have to be renovated every year, every two years? If an appointment expire, can the same person be appointed? If yes, Unlimited times? Consecutive times?

Reply:
I think it makes sense to now and then change the composition but changing the composition comes with a lot of complexity, especially with handling access to older reports. so imho I would for now lean towards simply making it indefinite (just like for the core team). I assume through change of focus and time available from enforcement team members, there will be some fluctuation anyway. and of course if there are concerns, Fabien can do it whenever he thinks its important.